src/modules/scene/reducer.ts

```diff
@@ -49,40 +48,78 @@ const INITIAL_STATE: SceneState = {
 const baseSceneReducer = (state: SceneState = INITIAL_STATE, action: SceneReducerAction): SceneState => {
   switch (action.type) {
     case CREATE_SCENE:
-    case PROVISION_SCENE:
-    case FIX_LEGACY_NAMESPACES_SUCCESS:
-    case SYNC_SCENE_ASSETS_SUCCESS:
     case LOAD_PROJECT_SCENE_SUCCESS: {
       const { scene } = action.payload
-
+      if (scene.sdk6) {
+        return {
+          ...state,
+          data: {
+            ...state.data,
+            [scene.sdk6.id]: {
+              ...scene,
+              sdk6: { ...scene.sdk6, components: { ...scene.sdk6.components }, entities: { ...scene.sdk6.entities } }
```

Comment:
why do we need this rewrite?

Reply:
it shouldn't, but I saw that the [original reducer](https://github.com/decentraland/builder/blob/master/src/modules/scene/reducer.ts#L62) does it that way, and I didn't want to risk that there's some mutation going on somewhere else that this cloning fixes 😅